docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -48,3 +48,69 @@ Then server `$hostname` supports secure protocols that $rule `$protocols`
 Scenario: Verify that the server supports TLSv1.3 and TLSv1.2 protocols
 Then server `vividus-test-site.herokuapp.com` supports secure protocols that contain `TLSv1.3,TLSv1.2`
 ----
+
+
+== Expressions
+
+
+=== URI Encoding
+
+`${encodeUriXyz(..)}` is a family of expressions that encode specific URI components (e.g. path, query)
+by percent encoding illegal characters, which includes non-US-ASCII characters, and also characters that
+are otherwise illegal within the given URI component type, as defined in
+https://www.ietf.org/rfc/rfc3986.txt[RFC 3986].
+
+[cols="3,2", options="header"]
+|===
+|Syntax
+|Description
+
+|`#{encodeUriUserInfo($userInfo)}`
+|Encode the given URI user info
+
+|`#{encodeUriHost($host)}`
+|Encode the given URI host
+
+|`#{encodeUriPath($path)}`
+|Encode the given URI path
+
+|`#{encodeUriPathSegment($segment)}`
+|Encode the given URI path segment
+
+|`#{encodeUriQuery($query)}`
+|Encode the given URI query
+
+|`#{encodeUriQueryParameter($queryParameter)}`
+|Encode the given URI query parameter
+
+|`#{encodeUriFragment($fragment)}`
+|Encode the given URI fragment
+|===
+
+==== Examples
+
+|===
+|Expression |Result
```

Comment:
|Expression
|Result

Reply:
it won't work without `options="header"`, so the current approach looks more concise